region/maridia/outer.json

```diff
@@ -2933,14 +2916,27 @@
                     "canSuitlessMaridia",
                     "h_canMaxHeightSpringBallJump",
                     "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      {"and": [
+                        "h_canMaxHeightSpringBallJump",
+                        {"or": [
+                          "canTrickyJump",
```

Comment:
`h_canMaxHeightSpringBallJump` includes canTrickyJump.

Reply:
It does, but I think it would be nice to make it it's own tech which doesn't rely on tricky jump after this.